### ISSUE TITLE
Is it possible to translate OpenVino person-detection-retail model to another framework, like tensorflow?

### ISSUE THREAD

**Issue by ya-smolin** [opencv/open_model_zoo#40]
I like pre-trained models provided by Intel OpenVino, but I want to run them using Tensorflow and GPU. For example, I am interested in people detection model written in Caffee. Is it legal to translate it to TesorFlow and use it in a commercial product? And does it makes sense? Maybe it is just better to train YOLO model for one class person?
Thank you for any help provided.

**Comment by snosov1** [comment]
> Is it legal to translate it to TesorFlow and use it in a commercial product?

Sorry, I don't think you can find any legal advise here in the open. I suggest to consult the legal department in your organization (that will, supposedly, produce the "commercial product"). The repo is licensed under Apache 2.0.

We start releasing some of the models in the original format (see https://github.com/opencv/training_toolbox_tensorflow and https://github.com/opencv/training_toolbox_caffe for example). It might better fit your needs.

> Maybe it is just better to train YOLO model for one class person?

I would strongly advise against training YOLO =) In our experience, SSD-like architectures provide a much better accuracy/peformance trade-off.
